Makefile

```diff
@@ -23,9 +23,6 @@ all: test lint
 dist:
 	for ex in $(TOOLS); do cd $$ex && make dist && cd -; done
 
-dockerhub:
-	for ex in $(TOOLS); do cd $$ex && make dockerhub && cd -; done
```

Comment:
It'd better to alter `TOOLS` than to drop the `dockerhub` target, no?

Reply:
It is not very easy to use makefile for travis builds. I was trying stuff.